examples/SkyTEM1D/Menindee/read_survey_data.jl

```diff
@@ -0,0 +1,177 @@
+
+    @info "reading $fname_specs_halt"
+    include(fname_specs_halt)
+    @assert size(d_LM, 2) == length(LM_times)
+    @assert size(d_HM, 2) == length(HM_times)
+    if !relerror
+        @assert size(d_LM, 2) == length(LM_noise)
+        @assert size(d_HM, 2) == length(HM_noise)
+        LM_noise[:] .*= units
+        HM_noise[:] .*= units
+    else
+        Ã_LM[:] .*= units
+        Ã_HM[:] .*= units
+    end
+    d_LM[:]     .*= units
+    d_HM[:]     .*= units
+    f = figure(figsize=figsize)
+    ax = Array{Any, 1}(undef, 4)
+    ax[1] = subplot(2,2,1)
+    nsoundings = size(soundings, 1)
+    plot_dLM = permutedims(d_LM)
+    plot_dLM[plot_dLM .<0] .= NaN
+    pcolormesh(1:nsoundings, LM_times, log10.(plot_dLM), shading="nearest")
+    xlabel("sounding #")
+    cbLM = colorbar()
+    cbLM.set_label("log d_LM")
+    ylabel("LM time s")
+    axLM = ax[1].twiny()
+    axLM.semilogy(LM_noise, LM_times)
+    axLM.set_xlabel("high alt noise")
+    ax[2] = subplot(2,2,3,sharex=ax[1], sharey=ax[1])
+    plot_dHM = permutedims(d_HM)
+    plot_dHM[plot_dHM .<0] .= NaN
+    pcolormesh(1:nsoundings, HM_times, log10.(plot_dHM), shading="nearest")
+    xlabel("sounding #")
+    cbHM = colorbar()
+    cbHM.set_label("log d_HM")
+    ylabel("HM time s")
+    ax[2].invert_yaxis()
+    axHM = ax[2].twiny()
+    axHM.semilogy(HM_noise, HM_times)
+    axHM.set_xlabel("high alt noise")
+    ax[3] = subplot(2,2,2, sharex=ax[1])
+    plot(1:nsoundings, zRx, label="Rx")
+    plot(1:nsoundings, zTx, label="Tx")
+    legend()
+    xlabel("sounding #")
+    ylabel("height m")
+    ax[3].invert_yaxis()
+    ax[4] = subplot(2,2,4, sharex=ax[1])
+    ax[4].plot(1:nsoundings, rRx)
+    xlabel("sounding #")
+    ylabel("rRx m")
+    plt.tight_layout()
+    if makesounding
+        s_array = Array{SkyTEMsoundingData, 1}(undef, nsoundings)
+        for is in 1:nsoundings
+            l, f = Int(whichline[is]), fiducial[is]
+            @info "read $is out of $nsoundings"
+            dlow, dhigh = vec(d_LM[is,:]), vec(d_HM[is,:])
+            if !relerror
+                ÃLM = sqrt.((multnoise*dlow).^2 + LM_noise.^2)
+                ÃHM = sqrt.((multnoise*dhigh).^2 + HM_noise.^2)
+            else
+                ÃLM, ÃHM = vec(Ã_LM[is,:]), vec(Ã_HM[is,:])
+            end
+            s_array[is] = SkyTEMsoundingData(rRx=rRx[is], zRxLM=zRx[is], zTxLM=zTx[is],
+                zRxHM=zRx[is], zTxHM=zTx[is], rTx=rTx, lowpassfcs=lowpassfcs,
+                LM_times=LM_times, LM_ramp=LM_ramp,
+                HM_times=HM_times, HM_ramp=HM_ramp,
+                LM_noise=ÃLM, HM_noise=ÃHM, LM_data=dlow, HM_data=dhigh,
+                sounding_string="sounding_$(l)_$f",
+                X=easting[is], Y=northing[is], Z=topo[is], fid=f,
+                linenum=l)
+        end
+        return s_array
+    end
+end
+
+read_survey_files("em_with_noise_calibration_lines.hdr",
+"/g/data/z67/nfm547/transD_GP/examples/SkyTEM1D/Menindee/coincidewithtempest.dat",
```

Comment:
Use relative paths here (i.e. just `coincidewithtempest.dat`, and run the script from within the `Menindee` folder), otherwise this won't work for other people who want to run this themselves